util/bigint/src/uint.rs

```diff
@@ -685,6 +687,17 @@ macro_rules! construct_uint {
 			}
 
 			#[inline]
+			fn to_hex(&self) -> String {
```

Comment:
perhaps, but we have `to_hex` implemented for other the other types and since this is bug fix PR i'd prefer to leave that for a second refactoring PR.